static/src/stylesheets/module/_adslot.scss

```diff
@@ -161,6 +154,13 @@
         padding-bottom: $gs-row-height / 2;
         padding-top: $gs-row-height / 2;
 
+        /* Oh horrible code, I'm looking at you. One day, you'll disappear.
+           One day, I'll fucking rip you apart.
+         */
+        &.ad-slot--h250 {
+            height: 250px;
```

Comment:
If the ad has a height of 230, are we happy showing an extra 20px gap underneath it? (AFAIK, the ad will just align to the top of its container)